shoplist/src/main/java/com/netguru/codereview/network/model/ShopListItemResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+data class ShopListItemResponse(
+    val itemId: String,
+    val name: String,
+    val quantity: Double
```

Comment:
It seems weird that the quantity of a shopping item could be a floating number. Consider replacing the unnecessary Double and declare the quantity value as an Int.